README.md

```diff
@@ -11,35 +11,29 @@
 
 ## What is PLANet?
 * PLANet is a time management library that automates the process of organizing your schedule.
-* PLANet takes your current schedule, your preferences and the different events and tasks that you want to add to that schedule and gives you the optimal schedule for your preferences.
+* PLANet takes your current schedule, preferences and different events and tasks that you want to add to that schedule and gives you the optimal schedule for your preferences.
 * PLANet lets you group together different tasks and time-slots according to category and importance so that your schedule suits your specific needs!
 * PLANet can split up different tasks so you can use your time more efficiently.
 
 ## How to use the library?
-The current upload format for this repository currently requires you to pull the repository into Android Studio and compile from there, though you are also welcome to take whichever files you want from this project and to add them to your code in any way you like :)
+The current upload format for this repository currently requires you to pull the repository into Android Studio and compile from there, though you are also welcome to take any file you want from the project and to use it in your code in any way you would like :)
```

Comment:
No need for "would like" the correct phrase is "any way you like"

Reply:
done